package.json

```diff
@@ -9,9 +9,12 @@
     "start:android:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'react-native run-android'",
     "start:ios": "concurrently -r 'react-native start' 'react-native run-ios'",
     "start:ios:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'react-native run-ios'",
+    "build:android:release": "cd android && ./gradlew assembleRelease",
     "link": "react-native link",
     "ios": "react-native run-ios",
     "android": "react-native run-android",
+    "android:release": "react-native run-android --variant=release",
+    "android:signkey": "cd android/app/ && keytool -genkey -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000 && cd -",
```

Comment:
I mean, change the `"android:signkey"` script by something like `"cd android/app/ && keytool -genkey -v -keystore RELEASE_KEYSTORE_PATH -alias ..."` where `RELEASE_KEYSTORE_PATH` is a local environment variable that points to the keystore file.

But I checked the `.gitignore` file and [every *.keystore are ignored](https://github.com/gitpoint/git-point/blob/master/.gitignore#L42) so @housseindjirdeh can put it here without problem. Actually I think it is already in the repo in the `android/keystore` directory :)